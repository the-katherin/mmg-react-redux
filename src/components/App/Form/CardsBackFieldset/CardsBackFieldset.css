.cards-back-fieldset__radios {
    display: grid;
    grid-auto-flow: column;
    grid-column-gap: 10px;

}

.cards-back-fieldset__radios img{
    border: 1px solid grey;
    width: 120px;
    height: 120px;
}