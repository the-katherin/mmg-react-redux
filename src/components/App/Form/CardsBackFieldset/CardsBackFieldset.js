import React from 'react';
import { cardsBack } from '../../../../assets/img';

const CardsBackFieldset = () => (
    <fieldset className='cards-back-fieldset' name='cardsBack'>

        <legend>Choose cards' back</legend>

        <div className='cards-back-fieldset__radios'>

            <div>
                <input type="radio" name='cardsBack' id='cardsback1' value="1" required />
                <label htmlFor="cardsback1">
                    <img src={cardsBack[1]} alt="first cards' back" />
                </label>
            </div>

            <div>
                <input type="radio" name='cardsBack' id='cardsback2' value="2" />
                <label htmlFor="cardsback2">
                    <img src={cardsBack[2]} alt="sec cards' back" />
                </label>
            </div>

            <div>
                <input type="radio" name='cardsBack' id='cardsback3' value="3" />
                <label htmlFor="cardsback3">
                    <img src={cardsBack[3]} alt="third cards' back" />
                </label>
            </div>

        </div>

    </fieldset>
);

export default CardsBackFieldset;