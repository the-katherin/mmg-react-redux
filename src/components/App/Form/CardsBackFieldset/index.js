import './CardsBackFieldset.css';

export { default } from './CardsBackFieldset';