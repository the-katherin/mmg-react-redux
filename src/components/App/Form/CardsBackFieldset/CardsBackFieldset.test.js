import React from "react";
import CardsBackFieldset from "./CardsBackFieldset";

describe("CardsBackFieldset", () => {
    test("match snapshot", () => {
        const wrapper = shallow(<CardsBackFieldset />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});