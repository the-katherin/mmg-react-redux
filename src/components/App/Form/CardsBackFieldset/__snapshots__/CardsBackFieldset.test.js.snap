// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardsBackFieldset match snapshot 1`] = `
<fieldset
  className="cards-back-fieldset"
  name="cardsBack"
>
  <legend>
    Choose cards' back
  </legend>
  <div
    className="cards-back-fieldset__radios"
  >
    <div>
      <input
        id="cardsback1"
        name="cardsBack"
        required={true}
        type="radio"
        value="1"
      />
      <label
        htmlFor="cardsback1"
      >
        <img
          alt="first cards' back"
          src="cardsback1.jpg"
        />
      </label>
    </div>
    <div>
      <input
        id="cardsback2"
        name="cardsBack"
        type="radio"
        value="2"
      />
      <label
        htmlFor="cardsback2"
      >
        <img
          alt="sec cards' back"
          src="cardsback2.jpg"
        />
      </label>
    </div>
    <div>
      <input
        id="cardsback3"
        name="cardsBack"
        type="radio"
        value="3"
      />
      <label
        htmlFor="cardsback3"
      >
        <img
          alt="third cards' back"
          src="cardsback3.jpg"
        />
      </label>
    </div>
  </div>
</fieldset>
`;
